src/patternfly/components/TextInputGroup/text-input-group.scss

```
@@ -35,12 +38,28 @@
   --pf-c-text-input-group__utilities--c-button--PaddingRight: var(--pf-global--spacer--xs);
   --pf-c-text-input-group__utilities--c-button--PaddingLeft: var(--pf-global--spacer--xs);
 
+  // input disabled style
+  --pf-c-text-input-group--m-disabled--Color: var(--pf-global--disabled-color--100);
+  --pf-c-text-input-group--m-disabled--BackgroundColor: var(--pf-global--disabled-color--300);
+  --pf-c-text-input-group--m-disabled--BorderColor: transparent;
+
   position: relative;
   display: flex;
+  color: var(--pf-c-text-input-group--Color);
```

Can you give this property a default value? You could also set `--pf-c-text-input-group--Color`, but you would have to set to `--pf-global--Color--100` or something instead of `inherit` since `inherit` is a special word and would apply to the custom property instead of being passed as the value to some other property when the custom property is used.

```suggestion
  color: var(--pf-c-text-input-group--Color, inherit);
```